scripts/release/generate_release_script.py

```diff
@@ -74,6 +78,8 @@ def beforeInclude(self, *args, **kwargs):
                         'fi',
                     ]
                     script = '\n'.join(lines)
+                elif 'Upload binary' in script or 'Upload source' in script:
```

Comment:
What's the purpose of this short circuit. Without knowing the context I'm a little concerned that it could be brittle. I think a comment explaining what scripts are being dropped by this would help put me at ease.

Reply:
:+1: b485bc6